lib/modules/ftp/weak_password/files/file_watcher_script.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+import json
+import time
+import datetime
+from watchdog.observers import Observer
+from watchdog.events import FileSystemEventHandler
+
+EXCLUDES = ['/dev']
+LOGFILE = '/root/logs/ohp_ftp_weak_password_logs.txt'
+
+
```

Comment:
Maybe follow the same method to write the constant variables in other files/classes as you did here?